ansible-ipi-install/roles/installer/tasks/main.yml

```diff
@@ -43,6 +43,8 @@
   tags: extramanifests
 - include_tasks: 55_customize_filesystem.yml
   tags: customfs
+- include_tasks: 57_check_vip_status.yml
```

Comment:
you need to go into main.yml and put this in the correct numerical order

Reply:
Just ordered the tasks as per the order

Reply:
Added the new logic, Thanks to @rlopez133  ; Can you guys please review the above logic if it works @e-minguez @iranzo

Reply:
@DirectedSoul1 can you squash the commits into one? thanks

Reply:
sure @rlopez133 I will do it now.